sound/soc/sof/ipc4-topology.c

```diff
@@ -819,6 +837,133 @@ static void sof_ipc4_unprepare_copier_module(struct snd_sof_widget *swidget)
 	ida_free(&fw_module->m_ida, swidget->instance_id);
 }
 
+static int snd_sof_get_pcm_hw_params(struct snd_sof_dev *sdev, struct snd_pcm_hw_params *params,
+				     int *sample_rate, int *channel_count, int *bit_depth)
+{
+	int depth;
+
+	switch (params_format(params)) {
+	case SNDRV_PCM_FORMAT_S16_LE:
+		depth = 16;
+		break;
+	case SNDRV_PCM_FORMAT_S24_LE:
+		depth = 24;
```

Comment:
is depth for container bits size or valid sample bits ?

Reply:
> is depth for container bits size or valid sample bits ?

Both